Mathlib/Data/Nat/BitIndices.lean

```diff
@@ -0,0 +1,114 @@
+/-
+Copyright (c) 2024 Peter Nelson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Peter Nelson
+-/
+import Batteries.Data.List.Pairwise
+import Mathlib.Data.Nat.Bitwise
+import Mathlib.Algebra.BigOperators.Ring.List
+import Mathlib.Algebra.Order.BigOperators.Group.List
+import Mathlib.Algebra.Star.Order
+import Mathlib.Data.List.Sort
+import Mathlib.Algebra.Order.Sub.Defs
```

Comment:
Please keep imports sorted if possible

Reply:
Can you point to a reference that this is the preferred style? It seems to me like either this should be in the style guide and enforced by a linter (as in `mathlib.lean`) or up to the author.